src/parser/monk/mistweaver/modules/spells/Vivify.js

```diff
@@ -27,20 +27,32 @@ class Vivify extends Analyzer {
   gustsHealing = 0;
   lastCastTarget = null;
   remDuringManaTea = 0;
+  countForGusts = false;
```

Comment:
Is this variable used anywhere else?

Reply:
This is an artifact of other attempts to fix the mastery issue